src/plugins/index_pattern_field_editor/public/components/delete_field_provider.tsx

```diff
@@ -8,14 +8,14 @@
 
 import React, { useCallback, useRef, useEffect } from 'react';
 
-import { IndexPattern } from '../shared_imports';
+import { DataView } from '../shared_imports';
 import { OpenFieldDeleteModalOptions } from '../open_delete_modal';
 import { CloseEditor } from '../types';
 
 type DeleteFieldFunc = (fieldName: string | string[]) => void;
 export interface Props {
   children: (deleteFieldHandler: DeleteFieldFunc) => React.ReactNode;
-  indexPattern: IndexPattern;
+  indexPattern: DataView;
```

Comment:
WOuld it be possible to pull out some of these small refactorings into a PR that goes directly into the master branch? Would help reduce the size of the PR, even if by a small amount, every file that doesn't show up here helps reviewers focus on the important changes.

Reply:
I reverted the change (https://github.com/elastic/kibana/pull/110226/commits/e2793c07fbf42fd2ffcbd01c519c479ef3259b54)